src/app/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css?family=Ubuntu:400,700');
+
+
+
```

Comment:
Over het algemeen gezien:
- Hanteer in CSS kebab casing in classnames en id's ( hier meer hierover: https://stackoverflow.com/questions/7560813/why-are-dashes-preferred-for-css-selectors-html-attributes )
- Gebruik zo min mogelijk ID's om te stylen, maar alleen om javascript te triggeren.